packages/flutter/lib/src/widgets/table.dart

```diff
@@ -438,33 +504,100 @@ class _TableElement extends RenderObjectElement {
 /// as the [child].
 class TableCell extends StatelessWidget {
   /// Creates a widget that controls how a child of a [Table] is aligned.
-  const TableCell({super.key, this.verticalAlignment, required this.child});
+  const TableCell({
+    super.key,
+    this.colSpan = 1,
+    this.rowSpan = 1,
+    this.verticalAlignment,
+    required this.child,
+  }) : assert(colSpan >= 1, 'The colSpan of a TableCell must be at least 1.'),
+       assert(rowSpan >= 1, 'The rowSpan of a TableCell must be at least 1.');
+
+  /// Internal constructor used for [TableCell.none].
+  const TableCell._none()
```

Comment:
Is there a reason for introducing this as a private constructor? Why not make it .none()?